reladomographql/src/test/resources/test-schema.graphqls

```diff
@@ -0,0 +1,541 @@
+
+    orderItemById( id: Int! ): OrderItem
+    orderItems(findMany: OrderItemFinder filter: OrderItemFinder where: OrderItemFinder order_by: OrderItemOrderBy limit: Int): [OrderItem]
+    orderItem_aggregate(findMany: OrderItemFinder filter: OrderItemFinder where: OrderItemFinder order_by: OrderItemOrderBy limit: Int): [OrderItemAggregate]
+
+    tamsMithraAccountById( accountNumber: String! ): TamsMithraAccount
+    tamsMithraAccounts(findMany: TamsMithraAccountFinder filter: TamsMithraAccountFinder where: TamsMithraAccountFinder order_by: TamsMithraAccountOrderBy limit: Int): [TamsMithraAccount]
+    tamsMithraAccount_aggregate(findMany: TamsMithraAccountFinder filter: TamsMithraAccountFinder where: TamsMithraAccountFinder order_by: TamsMithraAccountOrderBy limit: Int): [TamsMithraAccountAggregate]
+
+}
+
+
+#################### com.gs.fw.common.mithra.test.domain.TestBalanceNoAcmap ##################
+type TestBalanceNoAcmap
+{
+    accountId: String
+    productId: Int
+    positionType: Int
+    quantity: Float
+    businessDateFrom: Timestamp
+    businessDateTo: Timestamp
+    processingDateFrom: Timestamp
+    processingDateTo: Timestamp
+    businessDate: Timestamp
+    processingDate: Timestamp
+    account: TamsMithraAccount
+}
+
+type TestBalanceNoAcmapAggregate
+{
+    accountId: String
+    productId: Int
+    varianceSampleOfProductId: Int
+    maxOfProductId: Int
+    avgOfProductId: Int
+    minOfProductId: Int
+    standardDeviationPopulationOfProductId: Int
+    sumOfProductId: Int
+    variancePopulationOfProductId: Int
+    standardDeviationSampleOfProductId: Int
+    positionType: Int
+    varianceSampleOfPositionType: Int
+    maxOfPositionType: Int
+    avgOfPositionType: Int
+    minOfPositionType: Int
+    standardDeviationPopulationOfPositionType: Int
+    sumOfPositionType: Int
+    variancePopulationOfPositionType: Int
+    standardDeviationSampleOfPositionType: Int
+    varianceSampleOfQuantity: Float
+    maxOfQuantity: Float
+    avgOfQuantity: Float
+    minOfQuantity: Float
+    standardDeviationPopulationOfQuantity: Float
+    sumOfQuantity: Float
+    variancePopulationOfQuantity: Float
+    standardDeviationSampleOfQuantity: Float
+    businessDateFrom: Timestamp
+    businessDateTo: Timestamp
+    processingDateFrom: Timestamp
+    processingDateTo: Timestamp
+    account: TamsMithraAccount
+}
+
+input TestBalanceNoAcmapInput
+{
+    accountId: String
+    productId: Int
+    positionType: Int
+    quantity: Float
+    businessDateFrom: Timestamp
+    businessDateTo: Timestamp
+    processingDateFrom: Timestamp
+    processingDateTo: Timestamp
+}
+
+input TestBalanceNoAcmapOrderBy
+{
+    accountId: AcsDesc
+    productId: AcsDesc
+    positionType: AcsDesc
+    quantity: AcsDesc
+    businessDateFrom: AcsDesc
+    businessDateTo: AcsDesc
+    processingDateFrom: AcsDesc
+    processingDateTo: AcsDesc
+}
```

Comment:
Because the NameOrderBy types like this, and the parameter is a single NameOrderBy rather than an array of them, I can't figure out how you'd specify you want to order by several attributed in a particular order (`productId asc, accountId asc` rather than `accountId asc, productId asc`)

Reply:
right now it only supports a single GroupBy attribute. I will try to implement improvement in a few days.